cmd/estuary-shuttle/main.go

```diff
@@ -476,9 +504,9 @@ func main() {
 				var received uint64
 
 				for _, xfer := range txs {
-					byState[xfer.Status()]++
-					sent += xfer.Sent()
-					received += xfer.Received()
+					byState[xfer.Status]++
```

Comment:
why this change?

Reply:
The format of the data returned by TransfersInProgress [has changed slightly](https://github.com/application-research/filclient/pull/60/files#diff-1dfb05953decb16f2625b57212a3b0c11b22110804a0b24c4ec2cba68162c3f2R940) - it now returns a ChannelState object which has a field called Status instead of a method.

Reply:
cool, I see why the filclient go mod update